src/integrationTest/codewhisperer/referenceTracker.test.ts

```diff
@@ -0,0 +1,120 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as codewhispererClient from '../../codewhisperer/client/codewhisperer'
+import { ConfigurationEntry } from '../../codewhisperer/models/model'
+import { setValidConnection } from './util'
+import { RecommendationHandler } from '../../codewhisperer/service/recommendationHandler'
+import { createMockTextEditor, resetCodeWhispererGlobalVariables } from '../../test/codewhisperer/testUtil'
+import { invokeRecommendation } from '../../codewhisperer/commands/invokeRecommendation'
+
+const leftContext = `InAuto.GetContent(
```

Comment:
If there is a model deployment, the response may not contain code reference because the recommendation will change. Is there any mitigation? If no, please be prepared.

Reply:
Then we need to change, 1) the tests are not required for github approval flow 2) we can mark it as skipped until we posted a fix for manual runs

Reply:
Thanks, I updated the `referenceTracker` tests to automatically skip for now and added the following comment to the top of the `referenceTracker.test.ts` file:

```
/*
New model deployment may impact references returned. 

These tests:
    1) are not required for github approval flow 
    2) will be auto-skipped until fix for manual runs is posted.
*/
```